README.md

```diff
@@ -137,24 +278,10 @@ acm-pca-cert-generator --key-type RSA --key-length 2048 --subject-c "GB" \
 --subject-st "Greater London" --subject-l "London" --subject-o "My Company" \
 --subject-ou "IT Department" --subject-cn "myfqdn.example.com" \
 --subject-emailaddress "me@example.com" \
---ca-arn "arn:aws:acm-pca:us-east-1:012345678901:certificate-authority/506a130d-8519-45dc-903d-2a30709d6a33" \
+--ca-arn "arn:aws:acm-pca:us-east-1:012345678901:certificate-authority/a1a1a1a1a1a1a1a1a1a1a1" \
```

Comment:
I actually preferred my made up example, simply because whilst not real, the ARN was at least in a valid format that looks like what it should in a real scenario.

Reply:
Done!